roles/container-storage-setup/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- block:
+    - name: create the docker-storage config file
+      template:
+        src: docker-storage-setup-overlayfs.j2
+        dest: /etc/sysconfig/docker-storage-setup
+        owner: root
+        group: root
+        mode: 0644
+  when:
```

Comment:
This `when` check should be moved to the very bottom of the file to only do the docker check/restart if "ephemeral volume" isn't used